src/sites/twitch-twilight/modules/css_tweaks/styles/hide-gift-subscribe-button.scss

```diff
@@ -0,0 +1,3 @@
+button[data-test-selector="gift-subscribe-button"] {
```

Comment:
As indicated in the previous comment, this should be implemented with `CLASSES` and not its own CSS blob.

Reply:
Also this selector is fairly non-specific if it's supposed to only match viewer cards.

Reply:
i'm not quite sure how to make it more specific, because from what i read on  another pull request we shouldn't use classes from twitch so the only useful attribute i found was  `data-test-selector="gift-subscribe-button`.

Reply:
I'm talking about parent elements.
```css
.viewer-card button[data-test-selector="gift-subscribe-button"]
```